bin/cml-runner.js

```diff
@@ -43,6 +43,7 @@ const shutdown = async (opts) => {
   const { name, workdir = '' } = opts;
   const tf_path = workdir;
 
+  console.log('Destroying runner...');
```

Comment:
```suggestion
  console.log(JSON.stringify({level: "info", status: "terminated"}));
```

Could we standardize on the machine-readable JSONL format used [here](https://github.com/iterative/cml/blob/81e8a363eb300306c888a77bdeb4d71179f09203/bin/cml-runner.js#L208) for consistency? It's uglier, but also a lot easier to process in a consistent way.

Reply:
Ok I will change it. But we are adding later on more logs that breaks the JSON output. We have to fix that

Reply:
No worries, the provider is smart enough to pick only the JSONL lines. Anyhow #22 will have the last word.